src/framework/preferences/js/Panels.js

```diff
@@ -692,6 +692,93 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
         }
     });
 
+    /********************************************************
+     * A base grade for styled radio Button adjuster panels *
+     ********************************************************/
+
+    fluid.defaults("fluid.prefs.panel.styledRadioButtonAdjuster", {
+        gradeNames: ["fluid.prefs.panel"],
+        mergePolicy: {
+            "controlValues.theme": "replace",
```

Comment:
I wonder if the word `theme` meant the theme/contrast being selected. If so, might be better to use a more generic word.